azavea/azavea-branding-guide
# Add new Cicero logos and tagline

## Overview
Adds new Cicero logo assets and updates the tagline.

### Demo
<img width="768" alt="Screen Shot 2020-12-20 at 3 20 43 PM" src="https://user-images.githubusercontent.com/5672295/102711962-038e6e80-42d7-11eb-86ac-fde3464ad68b.png">
<img width="1680" alt="Screen Shot 2020-12-20 at 3 20 55 PM" src="https://user-images.githubusercontent.com/5672295/102711967-06895f00-42d7-11eb-9520-ef4f53de5a34.png">


## Testing Instructions
* git pull
* Go to the most recent release note at the bottom of the sidebar
* Click the links to make sure they work and view the updated Cicero section in the **Products page**.
* Click the Download buttons. Ensure that the name of the zip looks right, and that you get new logos in the download.
* Navigate to the Logo Downloads page.
* Click the Download buttons for each here. Ensure that the name of the zip looks right.

Connects https://github.com/azavea/azavea-marketing/issues/112
